src/components/capacity-breakdown/capacity-breakdown-card.tsx

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright contributors to the ibm-storage-odf-console project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Select, SelectProps } from '@patternfly/react-core';
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+  usePrometheusPoll,
+} from '@console/dynamic-plugin-sdk/internalAPI';
+import { BreakdownCardBody } from '../breakdown-card/breakdown-body';
+import { getStackChartStats, sortInstantVectorStats } from '../breakdown-card/utils';
+import { getSelectOptions } from '../breakdown-card/breakdown-dropdown';
+import './capacity-breakdown-card.scss';
+import { humanizeBinaryBytes } from '../../humanize';
+import { BreakdownQueryMapODF } from '../../constants/queries';
+import { PROJECTS, STORAGE_CLASSES, PODS } from '../../constants/index';
+import { 
+  getInstantVectorStats,
+} from '../../selectors/promethues-utils';
+import { parseProps } from '../../selectors/index';
+
+const keys = [PROJECTS, STORAGE_CLASSES, PODS];
```

Comment:
nits (not anything important): variable name as `dropdownKeys` or `dropdownItems` could be used for better readability.